.github/workflows/create-score-version.yml

```diff
@@ -115,11 +76,64 @@ jobs:
           mkdir -p /usr/local/bin
           cp tippecanoe /usr/local/bin/tippecanoe
           tippecanoe -v
+          
+      # Generate score and map tiles
+      - name: Generate Score
+        run: |
+          poetry run python3 data_pipeline/application.py score-full-run
+      - name: Generate Score Post
+        run: |
+          poetry run python3 data_pipeline/application.py generate-score-post -s aws
+      - name: Generate Score Geo
+        run: |
+          poetry run python3 data_pipeline/application.py geo-score
       - name: Generate Tiles
         run: |
           poetry run python3 data_pipeline/application.py generate-map-tiles
+          
+      # Run basic smoke tests on score and map tiles
+      - name: Confirm we generated the version of the score we think we did
+        if: ${{ env.J40_VERSION_LABEL_STRING != 'beta' }}
+        run: |
+          grep "Identified as disadvantaged due to tribal overlap" data_pipeline/data/score/downloadable/* > /dev/null
+      - name: Confirm we generated the version of the score we think we did
+        if: ${{ env.J40_VERSION_LABEL_STRING == 'beta' }}
+        run: |
+          grep -v "Identified as disadvantaged due to tribal overlap" data_pipeline/data/score/downloadable/* > /dev/null
+      - name: Run smoketest
+        if: ${{ env.J40_VERSION_LABEL_STRING != 'beta' }}
+        run: |
+          poetry run pytest data_pipeline/ -m smoketest
+      
+      # Deploy to AWS
+      - name: Deploy Score to Geoplatform AWS
+        run: |
+          poetry run s4cmd put ./data_pipeline/data/score/csv/ s3://justice40-data/data-versions/${{env.J40_VERSION_LABEL_STRING}}/data/score/csv --recursive --force --API-ACL=public-read
+          poetry run s4cmd put ./data_pipeline/files/${{env.J40_VERSION_LABEL_STRING}}/ s3://justice40-data/data-versions/${{env.J40_VERSION_LABEL_STRING}}/data/score/downloadable --recursive --force --API-ACL=public-read
```

Comment:
It seems that the [`files`](https://github.com/usds/justice40-tool/tree/main/data/data-pipeline/data_pipeline/files) folder is for _static files_ (ie files that are to be downloadable from the client and not created by the pipeline) while the [`downloadable`](https://github.com/usds/justice40-tool/tree/main/data/data-pipeline/data_pipeline/data/score/downloadable) folder is for files that _are_ created by the pipeline. These folder names don't make it obvious to me that this is how they operate. I wonder if changing them makes sense?

These are the _static files_ _for 1.0_:

[1.0-cejst-technical-support-document.pdf](https://static-data-screeningtool.geoplatform.gov/data-versions/1.0/data/score/downloadable/1.0-cejst-technical-support-document.pdf)
[CEQ-CEJST-Instructions.pdf](https://static-data-screeningtool.geoplatform.gov/data-versions/1.0/data/score/downloadable/CEQ-CEJST-Instructions.pdf)
[1.0-communities-list.pdf](https://static-data-screeningtool.geoplatform.gov/data-versions/1.0/data/score/downloadable/1.0-communities-list.pdf)
[1.0-climate-and-environmental-burden-tool-comparison.pdf](https://static-data-screeningtool.geoplatform.gov/data-versions/1.0/data/score/downloadable/1.0-climate-and-environmental-burden-tool-comparison.pdf)

The files currently in `files` are for `beta` only ( I think). Matter of fact they may not be used anywhere anymore. I believe these files are strictly for the client and since the client will not likely revert to the beta version, maybe it's safe to remove these and replace with the 1.0 static files? In which case my next suggestion of changing `files` to `files/beta` and `files/1.0` may be moot.

Lastly, given that these CLI are PUTs, _not_ having these 1.0 files won't break anything, however if S3 bucket contents are lost for any reason, maybe it's a good thing to have these 1.0 static files in source control. 

What do you think?

Reply:
Yes, and. 🤣 Just for the historical record, _some_ of the files in the files folder are repackaged into {version}-data-documentation.zip, and are uploaded to S3 (e.g. communities-list.pdf and cejst-technical-support-document.pdf).